tools/db_bench_tool.cc

```diff
@@ -67,6 +68,7 @@
 #include "util/xxhash.h"
 #include "utilities/blob_db/blob_db.h"
 #include "utilities/merge_operators.h"
+#include "utilities/persistent_cache/block_cache_tier.h"
```

Comment:
Not in this diff, but I think we should consider to move db_bench to utilities, because it depends on too many components under utilities.